railties/lib/rails/application/configuration.rb

```diff
@@ -202,6 +202,10 @@ def load_defaults(target_version)
           ActiveSupport.utc_to_local_returns_utc_offset_times = true
         when "6.2"
           load_defaults "6.1"
+
+          if respond_to?(:active_support)
+            active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
```

Comment:
Let's add this config to https://github.com/rails/rails/blob/96db850d5197790e5e2c9f64309945d26185a64d/railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_2.rb.tt.

I think we either need to document how to rotate existing keys to the new default, or tell upgrade users to explicitly set the SHA1.

Reply:
@rafaelfranca I've added this, let me know how this is looking then :smile:.